### ISSUE TITLE
[Bug][a11y] Some elements from uBlock Origin's settings page aren't read 

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#8203]
## Prerequisites
1. Make sure uBlock Origin is installed.
2. Turn Talkback on.

## Steps to reproduce
1. Open a page, open the three-dot menu, and tap on the uBlock Origin's icon.
2. In the uBlock Origin's settings page, go through each element.

### Expected behavior
All elements are read or presented.

### Actual behavior
The first two elements under the enable/disable button aren't read/presented, and cannot be selected at all.

### Device information
* Android device: Google Pixel (Android 10)
* Fenix version: 2/7 Nightly build

![20200207_Trim1](https://user-images.githubusercontent.com/40694158/74031830-354a5f80-49bb-11ea-94ce-9e5a168a13ac.gif)

**Comment by csadilek** [comment]
This would need to be addressed in the add-on itself //cc @brampitoyo @kewisch

**Comment by softvision-miralobontiu** [comment]
I will close this issue based on the fact that the actual content of the addon can only be manipulated by the author.
